src/initialMarker/initialMarkerChunks.js

```diff
@@ -0,0 +1,36 @@
+const sortChunks = require('./sortChunks');
+
+const getMarkedChunks = (bundleArr, initialChunks, initialPrefix, isServerChunk, serverPrefix) => {
+    
+    let initialArr = [], secondaryArr = [], serverArr = [];
+
+    bundleArr.map(item => {
+        if (isServerChunk && item.chunkNames[0] === 'server') {
```

Comment:
if that is isServerChunk, then no need to check chunkNames, we need to mark all of them as server